src/utils/buildProject.js

```diff
@@ -0,0 +1,47 @@
+import joi from 'joi';
+
+export default function buildAdkAsync(
+  appJson : Object,
+  options: {
+    current?: boolean,
+    mode?: string,
+    platform?: string,
+    expIds?: Array<string>,
+    type?: string,
+    releaseChannel?: string,
+    bundleIdentifier?: string,
+    publicUrl?: string,
+    sdkVersion?: string,
+  } = {}
+) {
+
+  const schema = joi.object().keys({
```

Comment:
I believe the joi validation is there to provide users immediate and explicit feedback on project configuration issues.  Would any of these be supplied by end users working on snack, or just whatever projects are using the sdk?

Reply:
(for now)